modules/nw-multus-add-pod.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+
+[id="nw-multus-add-pod_{context}"]
+= Adding a Pod to an additional network
+
+You can add a Pod to an additional network. {product-title} will continue using
+the primary network for normal cluster related network traffic.
+
+.Prerequisites
+
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* You must log in to the cluster.
+
+.Procedure
+
+To add a Pod to an additional network, complete the following steps:
+
+. Edit the Pod resource definition. If you are editing an existing Pod, run the
+following command to edit its definition in the default editor. Replace `<name>`
+with the name of the Pod to edit.
++
+----
+$ oc edit pod <name>
+----
+
+. In the Pod resource definition, include the `k8s.v1.cni.cncf.io/networks`
+parameter in the Pod `metadata` mapping. The `k8s.v1.cni.cncf.io/networks`
+accepts a comma separated string of one or more `AdditionalNetworkDefinition`
+names:
++
+[source,yaml] 
+----
+metadata:
+  annotations:
+    k8s.v1.cni.cncf.io/networks: <network>[,<network>,...] <1>
+----
+<1> Replace `<network>` with the name of the additional network to associate
+with the Pod. To specify more than one additional network, separate each network
+with a comma. Do not include any whitespace between the comma. If you specify
+the same additional network multiple times, the Pod will receive multiple IP
+addresses on that network.
```

Comment:
> the Pod will receive multiple IP addresses on that network

I think most of the time this does make sense and is intuitive, but, there are possibilities where you attach interfaces that don't have Layer 3 connectivity (and therefore don't have IP addresses) it may be more technically correct to say:

> that Pod will have multiple network interfaces attached to that network.